CHANGELOG.md

```diff
@@ -3,6 +3,48 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.16.0
+_Aug 11, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Improve quick filtering with row grouping (#5701) @alexfauquette
+- 📚 Documentation improvements
+- 🐞 Bugfixes
```

Comment:
Did not noticed that before but sometime we write `Bug fixes` and sometime `Bugfixes`
I personally prefer the format but let's pick one and stick to it

Reply:
32 instances of `bugfixes` and 5 instances of `bug fixes`. 🤔 
I guess it's safe to say that we want to `stick` with `bugfixes`? 😄